content/docs/studio/user-guide/projects-and-experiments/run-experiments.md

```diff
@@ -79,24 +79,47 @@ on `Commit changes`.
 
 ![](https://static.iterative.ai/img/studio/cml_commit.png)
 
-At this point, the new experiment appears in the project's experiment table. If
-you just committed to a new branch, then a new pull request will also have been
-created from the new branch to the base branch.
+## What happens after you submit a new experiment
+
+### Git commit (and pull request) are created
+
+The new experiment appears in the project's experiment table. If you had
+specified a new branch to commit the changes to, then a new pull request will
+also have been created from the new branch to the base branch.
```

Comment:
So is an experiment created? Is a commit created? This may be confusing. I don't think we should call experiments commits until they are "made persistent" (but we should probably also avoid that term).

> Also the grammatical number is unclear i.e. is it "Git commit (or PR) is" -or- "Git commit(s) and PR(s) are" ?

Reply:
It's 1 Git commit and 0 or 1 PR. Not sure what's the best way to say this.

Reply:
OK, the grammatical number is correct then (sounds a bit strange but it's a very minor observation, moving on...)

I see the text is clarified. Thanks, resolving.